apps/api/src/app/bridge/usecases/get-bridge-status/get-bridge-status.usecase.ts

```diff
@@ -9,7 +9,10 @@ const axiosInstance = axios.create();
 export class GetBridgeStatus {
   async execute(command: GetBridgeStatusCommand): Promise<HealthCheck> {
     try {
-      const response = await axiosInstance.get<HealthCheck>(`${command.bridgeUrl}?action=health-check`, {
+      const bridgeActionUrl = new URL(command.bridgeUrl);
+      bridgeActionUrl.searchParams.set('action', 'health-check');
```

Comment:
Thanks for refactoring this to use the `URL` API. We can additionally remove the hardcoded strings here with the following imports, creating a more intentional coupling between the Framework and API.

```typescript
import { PostActionEnum, GetActionEnum, HttpQueryKeysEnum } from '@novu/framework';
```

usage to replace (here and `preview-step.usecase`):
`'action'` -> `PostActionEnum.ACTION`
`'health-check'` -> `GetActionEnum.HEALTH_CHECK`
`'preview'` -> `PostActionEnum.PREVIEW`
`'workflowId'` -> `HttpQueryKeysEnum.WORKFLOW_ID`
`'stepId'` -> `HttpQueryKeysEnum.STEP_ID`

Reply:
Fixed!